ethereumetl/service/token_transfer_extractor.py

```diff
@@ -40,7 +40,7 @@ def extract_transfer_from_log(self, receipt_log):
             # This is normal, topics can be empty for anonymous events
             return None
 
-        if topics[0] == TRANSFER_EVENT_TOPIC:
+        if (topics[0]).casefold() == TRANSFER_EVENT_TOPIC:
```

Comment:
`if isinstance(topics[0], str) and (topics[0]).casefold() == TRANSFER_EVENT_TOPIC` to avoid null pointer error

Reply:
Why there would be null pointer?

Reply:
My bad, indeed it's checked a few lines above

Reply:
@ytrezq I ll let you backport it to Polygon-etl